core-bundle/tests/Session/Attribute/ArrayAttributeBagTest.php

```diff
@@ -47,7 +47,6 @@ public function testCanReadTheOffset(): void
 
         $this->assertSame('bar', $bag['foo']);
         $this->assertSame(['baz' => 'foo'], $bag['bar']);
-        $this->assertSame('foo', $bag['bar']['baz']);
```

Comment:
Why?

Reply:
Because of the assertion above. If `$bag['bar']` is `['baz' => 'foo']`, `$bag['bar']['baz']` can never be anything else but `'foo'`.

Reply:
In general that's only true for arrays, though, not for classes implementing `ArrayAccess`.

Reply:
Is it true in this specific case? Because if not, I can add the line again with a `@phpstan-ignore` comment.

Reply:
Yes, here it's fine to drop the line. :+1: